manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml

```diff
@@ -1,19 +1,38 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  name: nfd.v4.2.0
+  namespace: placeholder
   annotations:
     capabilities: Basic Install
-    categories: "Configuration"
-    description: This software enables node feature discovery for Kubernetes. It detects hardware features available on each node in a Kubernetes cluster, and advertises those features using node labels.
+    categories: "Database"
+    
     provider: Red Hat
     support: Red Hat
     containerImage: 
     createdAt: 2019-05-30T00:00:00Z
     certified: "false"
     repository: https://github.com/openshift/cluster-nfd-operator
-  name: nfd.v4.2.0
-  namespace: placeholder
+    alm-examples: |-
```

Comment:
This was added to pass the OLM test?

Reply:
Yes, and all other changes were added to pass operator-courier tests.